english/add_music.md

```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```

Comment:
I just did a search - you first have to clone to your local machine in order to create a directory, since github only allows you to create files on the web site?  I'll try that.